src/services/api/index.js

```diff
@@ -18,6 +18,7 @@ import songs from 'services/api/songs'
 import stats from 'services/api/stats'
 import stories from 'services/api/stories'
 import storyPages from 'services/api/storyPages'
+import vimeoThumbnail from 'services/api/videoLinkThumbnails'
```

Comment:
typo? vimeo vs video

Reply:
That specific API is to get the thumbnail from Vimeo videos only. The `videoLinkThumbnails` file could be used to add additional requests for other services in the future if needed.